ARCharts/ARBarChart+Bar.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  ARBarChartBar.swift
+//  ARCharts
+//
+//  Created by Christopher Chute on 7/25/17.
+//  Copyright © 2017 Boris Emorine. All rights reserved.
+//
+
+import SceneKit
+import UIKit
+
+extension ARBarChart {
+    
+    public class Bar: SCNNode {
+        
+        public var series: Int = 0
```

Comment:
The default value should probably not be `0` since it's a valid value. Can't we just not have a default value, and since they're passed in with the initializer, we shouldn't need to make them optional.